e2e/setup-local-fdb-cluster.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Set up local FDB non-HA cluster on a 4-node kind k8s cluster
```

Comment:
We should add a note here that this only works on a x64 machine as FDB doesn't provide arm64 Linux binaries yet.